build/patches/Add-an-always-incognito-mode.patch

```diff
@@ -605,6 +567,61 @@ diff --git a/chrome/android/java/src/org/chromium/chrome/browser/webapps/WebappI
      @Override
      public @ScreenOrientationLockType.EnumType int getDefaultOrientation() {
          return mWebappExtras.orientation;
+diff --git a/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc b/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
+--- a/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
++++ b/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
+@@ -305,6 +305,9 @@ signin::IdentityManager* ChromeAutocompleteProviderClient::GetIdentityManager()
+ }
+ 
+ bool ChromeAutocompleteProviderClient::IsOffTheRecord() const {
++  if (profile_->GetPrefs()->GetBoolean(prefs::kAlwaysIncognitoEnabled)) {
```

Comment:
This is not really nice; it would be best that this check is done on the places where `ChromeAutocompleteProviderClient::IsOffTheRecord()` is called instead of here because otherwise the function should be renamed `IsOffTheRecordButNotAlways` (better add the condition on call sites)

Reply:
ok, you're right, I'll check, in the meantime we remove it.

Reply:
fixed in https://github.com/bromite/bromite/pull/1827/commits/c8fafdc03eb47bc6f6fd6107577830542c2f549a

I found the only necessary point. since I don't have the preferences available at that point, I added `IsAlwaysIncognitoEnabled()` to `AutocompleteProviderClient` and derivatives